### PR TITLE
Finalize 2.0.0

### PR DESCRIPTION
**Pull request by zonky2**
Finalize 2.0.0

### CODE REVIEW

File: contao/dca/tl_metamodel_filtersetting.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * This file is part of MetaModels/filter_tags.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline contao/dca/tl_metamodel_filtersetting.php]: Eh, NO?!

File: contao/languages/en/tl_metamodel_filtersetting.php [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * This file is part of MetaModels/filter_tags.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline contao/languages/en/tl_metamodel_filtersetting.php]: Nope, not either?!

File: src/MetaModels/Filter/Setting/Tags.php [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * This file is part of MetaModels/filter_tags.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline src/MetaModels/Filter/Setting/Tags.php]: Not even here!

File: src/MetaModels/Filter/Setting/TagsFilterSettingTypeFactory.php [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * This file is part of MetaModels/filter_tags.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline src/MetaModels/Filter/Setting/TagsFilterSettingTypeFactory.php]: You know the routine.

File: tests/bootstrap.php [hunk 5]
```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * This file is part of MetaModels/filter_tags.
+ * This file is part of MetaModels/attribute_translatedurl.
```
Comment (discordier) [inline tests/bootstrap.php]: And finally: NOPE!

**Comment by zonky2** [comment]
fixed! thx

**Comment by discordier** [comment]
All good now.

**Comment by zonky2** [comment]
phuuu...

**Pull request merged**